packages/manager/modules/pci/src/projects/project/notebooks/dashboard/general-information/general-information.html

```diff
@@ -253,43 +253,140 @@
                     data-term="{{:: 'pci_notebooks_general_information_resources_quantity' | translate }}"
                 >
                     <oui-tile-description>
-                        <span
+                        <!-- GPU-->
+                        <p
+                            class="mb-1 font-weight-bold"
                             data-ng-if="$ctrl.flavor.gpuInformation"
                             data-ng-bind=":: $ctrl.notebook.gpu + $ctrl.NOTEBOOK_MULTIPLY_SIGN +  ($ctrl.notebook.flavor | uppercase)"
-                        ></span>
-                        <span
+                        ></p>
+                        <p
+                            class="mb-1"
+                            data-ng-if="$ctrl.flavor.gpuInformation"
```

Comment:
**thought:** redundant `ngIf` (same ngIf used three time in a row).

Reply:
Changed: https://github.com/ovh/manager/pull/6676/commits/75b74a4f02f352f06b940843b4ae27a514df043b